_posts/2016-01-15-locations-organizations-rbac.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: post
+title: Locations, Organizations and RBAC
+date: 2016-01-15 14:25:45
+author: Christopher Pisano - The Advisory Board Company
+tags:
+- foreman
+- multi-tenancy
+guest_post: true
+excerpt: |
+  This will discuss our need to build a multi-tenant Foreman implementation and
+  the necessity of Organizations, Locations and RBAC to do so.
+---
+
+###Overview
+
+We are a fairly large company consisting of eight offices and three data
+centers. Between these eight offices and three data centers we have about six
+organizations and about 20 products made available to our members. Each of these
+organizations have developers and system administrators that help manage and
+maintain their servers. For this reason we needed to build Foreman in a way
+where everyone can manage their servers and keep Enterprise Technologies from
+being a bottleneck.
+
+###Locations and Organizations
+
+Locations was a no brainer for us. With eight offices and three data centers we
+definitely needed a way to classify the location of our hosts and in turn use
+that classification to help configure our hosts with certain location specific
+data. Enabling Locations in Foreman solved this for us since the location
+parameter can be used as part of Hiera. The main caveat here is that each host
```

Comment:
Perhaps an example of how you use the location parameter in hiera would be useful?